src/components/common/YourCompany/YourCompany.vue

```diff
@@ -8,7 +8,7 @@
     <!-- Business/Company Name + Name Request Applicant -->
     <div id="company-name-section" class="section-container" :class="{'invalid-section': invalidNameSection}">
       <v-row no-gutters class="mt-4">
-        <v-col cols="3">
+        <v-col cols="3" class="pr-2">
```

Comment:
A little bit of whitespace between the label (column) and data. Otherwise, if you make your browser narrower, they can run into each other.